x/concentrated-liquidity/msg_server.go

```diff
@@ -103,6 +103,14 @@ func (server msgServer) WithdrawPosition(goCtx context.Context, msg *types.MsgWi
 			sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
 			sdk.NewAttribute(sdk.AttributeKeySender, msg.Sender),
 		),
+		sdk.NewEvent(
+			types.TypeEvtWithdrawPosition,
+			sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
+			sdk.NewAttribute(sdk.AttributeKeySender, msg.Sender),
+			sdk.NewAttribute(types.AttributeLiquidity, msg.LiquidityAmount.String()),
+			sdk.NewAttribute(types.AttributeAmount0, amount0.String()),
+			sdk.NewAttribute(types.AttributeAmount1, amount1.String()),
+		),
```

Comment:
Please do not add this. This should already be emitted inside keeper

Reply:
My bad, I didn't notice that comment there! Reverted https://github.com/osmosis-labs/osmosis/pull/4695/commits/e8bf75ddaee6adde0f33952aa6d9eca756d195e4